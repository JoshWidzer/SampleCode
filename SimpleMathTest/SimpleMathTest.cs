using System;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using ApprovalTests;
using ApprovalTests.Reporters;

namespace SimpleMathTest
{
    [TestClass]
    [UseReporter(typeof(DiffReporter))]
    public class SimpleMathTest
    {
        [TestMethod]
        public void OneAndOneIsTwo()
        {
            var math = new SimpleMath();
            var result = math.Add(1, 1);
            Assert.AreEqual(2, result);
        }

        [TestMethod]
        public void TwoAndTwoIsNotThree()
        {
            var math = new SimpleMath();
            var result = math.Add(2, 2);
            Assert.AreNotEqual(3, result);
        }

        [TestMethod]
        public void First10Fibonacci()
        {
            var math = new SimpleMath();
            var calculator = new Calculator(math);
            var result = calculator.GetFibonacci(10);
            Approvals.VerifyAll(result, "Index: ");
        }
    }
}
