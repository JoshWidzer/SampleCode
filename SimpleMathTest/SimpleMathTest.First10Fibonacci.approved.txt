Index: [0] = 1
Index: [1] = 1
Index: [2] = 2
Index: [3] = 3
Index: [4] = 5
Index: [5] = 8
Index: [6] = 13
Index: [7] = 21
Index: [8] = 34
Index: [9] = 55
Index: [10] = 89
